test/hurl/run.sh

```diff
@@ -1,6 +1,31 @@
 #!/bin/bash
 # Script to run unit tests on API with hurl
-#
+
+# default is to do all tests
+TEST_API=true
+TEST_GUI=true
+TEST_public=true
+
+for arg in "$@"
+do
+	case "$arg" in
+		--noapi)
+		TEST_API=false
+		;;
+		--nogui)
+		TEST_GUI=false
+		;;
+		--nopublic)
+		TEST_public=false
+		;;
+		*)
+		COOKIE=$( echo "$arg" | grep -c -- '--cookiefile=' )
```

Comment:
`$(grep -c -- '--cookiefile=' <<< "$arg")` saves an echo.

You can even write the following, bu tI think that may confuse.
`$(<<< "$arg"  grep -c -- '--cookiefile=' )`